app/services/pdf_text_extraction_service.py

```
@@ -448,6 +453,24 @@ def _extract_native_text(self, pdf_bytes: bytes) -> Dict:
+
+        This retains horizontal positions as whitespace so column-aware parsers
+        (e.g. the Epic MyChart card parser) can reconstruct columns. It is
+        materially more expensive than plain extraction, so it is only invoked
+        lazily for the parser that needs it.
+        """
+        layout_parts = []
+        with pdfplumber.open(io.BytesIO(pdf_bytes)) as pdf:
+            for page in pdf.pages:
+                try:
+                    layout_parts.append(page.extract_text(layout=True) or "")
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)